api/db/migrations/20240515151526_add-learner-import-to-sco-1d-organisation.js

```diff
@@ -0,0 +1,36 @@
+const up = async function (knex) {
+  const [feature] = await knex.select('id').from('features').where({ key: 'LEARNER_IMPORT' }).limit(1);
+  const organizationIdsSco1d = await knex.select('id').from('organizations').where({ type: 'SCO-1D' });
+  const [organizationLearnerImportFormat] = await knex
+    .select('id')
+    .from('organization-learner-import-formats')
+    .where({ name: 'ONDE' })
+    .limit(1);
+
+  if (!organizationLearnerImportFormat || organizationIdsSco1d.length === 0 || !feature) return;
```

Comment:
Supprimer la ligne : 
* si organizationIdsSco1d est vide, pas grave, ça itérera sur rien
* si organizationLearnerImportFormat ou feature ne sont pas présents, c'est un problème et ça ne doit pas passer silencieusement